README.md

```diff
@@ -1,76 +1,54 @@
-# Plugin Starter Template [![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template)
+# Mattermost/Microsoft Calendar Integration
```

Comment:
@aaronrothschild @mickmister We should probably rename the plugin to Microsoft Calendar (repo and all) before it's too late. `mscal` for command?

Reply:
We can't know for sure what functionality it will have in the future. Isn't it possible we also start handling other things in msoffice? i.e. email was received etc

Reply:
@levb Agreed, mscal for commands makes sense.  2/5 I guess we aren't doing `/office cal` ?